ods_ci/tests/Tests/650__distributed_workloads/test-run-dw-tests.robot

```diff
@@ -14,7 +14,8 @@ ${DW_JUNIT_FILE}          %{WORKSPACE=.}/junit.xml
 *** Test Cases ***
 Run distributed workloads sanity tests
     [Documentation]   Run tests located in Distributed Workloads repo downstream
-    [Tags]  Sanity
+    [Tags]  ODS-2511
+    ...     Sanity
 
     Skip If    "%{OPERATOR_TYPE}" != "RHODS Operator V2"
```

Comment:
Can you use the new keyword  `Skip If Component Is Not Enabled` introduced in #1010